PostHog/PostHogSDK.swift

```diff
@@ -599,6 +593,14 @@ let maxRetryDelay = 30.0
         )
 
         targetQueue?.add(posthogEvent)
+
+        #if os(iOS)
+            let surveysIntegration = installedIntegrations.compactMap {
+                $0 as? PostHogSurveyIntegration
+            }.first
+
+            surveysIntegration?.onEvent(event: posthogEvent.event)
```

Comment:
i think its easier to just cache the instance as we do with others.

Reply:
i think this can be a problem with multiple sdk instances though, unless you allow `surveysIntegration` to be installed for all instances, otherwise you might be calling the wrong one

Reply:
yeah i guess so, worth adding a comment on the posthog config (surveys: Boolean) though.